src/_settings.scss

```diff
@@ -214,6 +214,43 @@ $bgc-alert-basic-success: $c-white !default;
 // Button
 $bdrs-button: 0 !default;
 
+$c-button-primary-hover: $c-white !default;
+$c-button-primary: $c-white !default;
+$c-button-primary-icon: $c-white !default;
+$bgc-button-primary: $c-primary !default;
+$bgc-button-primary-hover: $c-primary-light !default;
+$bgc-button-primary-active: $c-primary-dark !default;
+
+$c-button-secondary: $c-primary !default;
+$c-button-secondary-hover: $c-primary !default;
+$c-button-secondary-icon: $c-primary !default;
+$bd-button-secondary: $c-primary !default;
```

Comment:
Since we are defining a border-color, the variable name should be `$bdc-button-secondary`.
Same for `$bd-button-accent` ...